cypress/e2e/buy-shirt.cy.ts

```diff
@@ -8,6 +8,7 @@ const paymentStepPage = new PaymentStepPage();
 const productsListPage = new ProductsListPage();
 const shippingStepPage = new ShippingStepPage();
 const shoppingCartPage = new ShoppingCartPage();
```

Comment:
I'm seeing a warning that this instance is not being used anywhere, please check

Reply:
Done